client/src/components/Layout/Navbar.tsx

```diff
@@ -0,0 +1,217 @@
+}
+
+interface NotificationProps {
+  titleAnchor: HTMLElement | null;
+  activeNotifications: ActiveNotification[];
+}
+
+const NotificationPopover: React.FC<NotificationProps> = ({ titleAnchor, activeNotifications }) => {
+  const classes = useStyles();
+  return (
+    <Popover
+      PaperProps={{
+        style: {
+          borderTop: '.25rem solid black',
+          borderRadius: '0px',
+          width: '300px',
+        },
+      }}
+      open={Boolean(titleAnchor)}
+      anchorEl={titleAnchor}
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'center',
+      }}
+      transformOrigin={{
+        vertical: 'top',
+        horizontal: 'center',
+      }}
+      className={classes.popover}
+    >
+      <MenuList autoFocusItem={Boolean(titleAnchor)}>
+        <MenuItem>
+          <Link className={classes.linkItem} to="/notifications">
+            {activeNotifications.map((notification) =>
+              notification.title !== null ? (
+                <Grid key={null} className={classes.notificationContainer}>
+                  <Typography className={classes.notificationTitle}>{notification.title}</Typography>
+                  <Typography className={classes.notificationSubtitle}>{notification.subtitle}</Typography>
+                  <Typography className={classes.notificationDate}>{notification.date}</Typography>
+                </Grid>
+              ) : (
+                <Grid key={null}>
+                  <Typography>No new notifications</Typography>
+                </Grid>
+              ),
+            )}
+          </Link>
+        </MenuItem>
+      </MenuList>
+    </Popover>
+  );
+};
+
+const Navbar: React.FC = () => {
+  const classes = useStyles();
+
+  const { logout } = useAuth();
+
+  const [profilePopoverAnchor, setProfilePopoverAnchor] = useState<null | HTMLElement>(null);
+  const [notificationsAnchor, setNotificationsAnchor] = useState<null | HTMLElement>(null);
+
+  const handleLogout = () => {
+    logout();
+  };
+
+  const profilePopover = (
+    <Popover
+      open={Boolean(profilePopoverAnchor)}
+      anchorEl={profilePopoverAnchor}
+      anchorOrigin={{
+        vertical: 'bottom',
+        horizontal: 'left',
+      }}
+      onClick={() => setProfilePopoverAnchor(null)}
+    >
+      <MenuList autoFocusItem={Boolean(profilePopoverAnchor)} onMouseLeave={() => setProfilePopoverAnchor(null)}>
+        <MenuItem onClick={() => setProfilePopoverAnchor(null)}>
+          <Link className={classes.linkItem} to="/profile">
+            Profile
+          </Link>
+        </MenuItem>
+        <MenuItem onClick={() => setProfilePopoverAnchor(null)}>
+          <Link className={classes.linkItem} onClick={handleLogout} to="/">
+            Logout
+          </Link>
+        </MenuItem>
+      </MenuList>
+    </Popover>
+  );
+
+  return (
+    <AppBar style={{ position: 'static' }}>
+      <Toolbar className={classes.navMain}>
+        <img src={logo} className={classes.logo} />
```

Comment:
```suggestion
        <img src={logo} alt='Logo' className={classes.logo} />
```
For web accessibility